StyleGuide.md

```diff
@@ -67,6 +67,11 @@ Other general guidance about language and tone:
 
     **Example:** Now that you have a database, user, and a table, run the following code to insert rows into the table.
 
+- Do not use the personal pronoun “we.” Instead, use:
+
+    - “CockroachDB” for when you are talking about something the _product_ does or supports.
+    - “Cockroach Labs” for when you are talking about something “we” (as the group of people developing CockroachDB) support or recommend.
```

Comment:
If we want to convey that we've actually tried things and are recommending as a company and you'll run into a problem.
Or if it's "strongly recommend" then it's CRL
Try to use "we recommend" more sparingly. Only use if it's a recommendation from eng or the field.